.github/workflows/playwright.yml

```diff
@@ -0,0 +1,26 @@
+name: Playwright Tests
```

Comment:
I think I just copied this template for GH Actions integration from the Playwright starter guide. It will probably need to be customized.